runbot_travis2docker/models/runbot_build.py

```diff
@@ -135,10 +137,9 @@ def job_10_test_base(self, cr, uid, build, lock_path, log_path):
         if not build.branch_id.repo_id.is_travis2docker_build:
             return super(RunbotBuild, self).job_10_test_base(
                 cr, uid, build, lock_path, log_path)
-        if not build.docker_image or not build.dockerfile_path \
-                or build.result == 'skipped':
-            _logger.info('docker build skipping job_10_test_base')
-            return MAGIC_PID_RUN_NEXT_JOB
+        skip = self.skip_check(build)
+        if skip:
+            return skip
```

Comment:
We have a skip status, we could use it in order to skip just the firts job method